llvm/test/MC/AArch64/armv9.6a-pcdphint.s

```diff
@@ -0,0 +1,12 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
+// RUN: llvm-mc -triple aarch64 -show-encoding -mattr=+pcdphint %s | FileCheck %s
```

Comment:
I think these tests should also check the instructions are rejected with `-mattr=+nopcdphint`

Reply:
Done